src/views/CharacterListView.js

```diff
@@ -27,9 +30,16 @@ class CharacterListView extends React.Component {
 
 // our mapStateToProps needs to have two properties inherited from state
 // the characters and the fetching boolean
+//the sub reducer is encompased inside the index reducers
+//destructering syntax to alias what's destructered ({ charsReducer: state })
+const mstp = ({ charsReducer: state }) => {
```

Comment:
What are some of the advantages/disadvantages to abbreviating methods in this way? How might you mitigate some of the disadvantages?